src/mgr/FunAction.cc

```diff
@@ -12,6 +12,7 @@
 #include "base/TextException.h"
 #include "comm/Connection.h"
 #include "globals.h"
+#include "ipc/RequestId.h"
```

Comment:
FYI: This and similar new `#include` statements in otherwise-unchanged `.cc` files are a side-effect of the (correct, IMO) `.h` header `#include` rule of thumb: Include only what the minimal user of the header would need. In other words, if some user of the header does not need `foo.h` to use the header, then the header should not `#include` `foo.h`.

In this particular case, that rule of thumb just keeps a few minor `#includes` out of the modified header files, but it may be very useful in more complex cases. Without it, most non-trivial source code files will eventually include "everything", slowing down compilation. complicating unit test cases, and sometimes even creating `#include` loops that need manual resolution.
 
The biggest problem with this rule of thumb, AFAICT, is that it cannot be tested automatically without a lot of effort. Generated tests would not fail because of an _extra_ header. And guessing which headers can be safely removed requires good "minimal user" test cases that cannot be easily generated. The test case that contains virtually nothing but the `#include` statement for the tested header is not a legitimate header "user" (and, IIRC, this case is automatically tested already for other reasons).